app/models/member.rb

```diff
@@ -51,6 +53,18 @@ def first_name
   def self.active
     Member.where(skip_meetings: false)
   end
+
+  def activate_account
```

Comment:
Rename to `activate!`. Trying to keep the language simple, you're activating a `Member`, not something else (an account).